test/files/jvm/t4283/Test.scala

```diff
@@ -1,4 +0,0 @@
-
-object Test extends App {
```

Comment:
this test was removed because a more exhaustive version of it exists in https://github.com/scala/scala/tree/2.12.x/test/files/run/t4283